README.md

```diff
@@ -39,41 +39,62 @@ we would like things to flow.
 ### Requirements
 
 * [Apache Maven 3.3.3+](https://maven.apache.org/install.html)
-* [Java 8+](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+* OpenJDK 8
 * Network access to https://repository.sonatype.org/content/groups/sonatype-public-grid
 
 Also, there is a good amount of information available at [Bundle Development](https://help.sonatype.com/display/NXRM3/Bundle+Development#BundleDevelopment-BundleDevelopmentOverview)
 
-NOTE: Version 0.0.2+ of Conan is only compatible with Nexus Repo 3.11.0 or greater due to adding Tree View Filtering. If 
-you need to use this plugin with an older version, please use 0.0.1
-
 ### Building
 
 To build the project and generate the bundle use Maven
 
-    mvn clean package -PbuildKar
+    mvn clean install
 
 If everything checks out, the bundle for Conan should be available in the `target` folder
 
-#### Build with Docker
+## Using Conan With Nexus Repository Manager 3
 
-`docker build -t nexus-repository-conan:0.0.6 .`
+[We have detailed instructions on how to get started here!](https://help.sonatype.com/repomanager3/formats/conan-repositories)
 
-#### Run as a Docker container
+## Compatibility with Nexus Repository Manager 3 Versions
 
-`docker run -d -p 8081:8081 --name nexus nexus-repository-conan:0.0.6` 
+The table below outlines what version of Nexus Repository Manager the plugin was built against:
 
-For further information like how to persist volumes check out [the GitHub repo for our official image](https://github.com/sonatype/docker-nexus3).
+| Plugin Version    | Nexus Repository Manager Version |
+|-------------------|----------------------------------|
+| v0.0.1            | <3.11.0                          |
+| v0.0.2+           | 3.11.0+                          |
+| v1.0.0+ In product| 3.22.0+                          |
```

Comment:
Trivial: Unless we don't intend to update this chart with further versions, I (trivially) think "v1.0.0+" can lose the + and be "v1.0.0".  I don't feel strongly.

Reply:
fd4cf415e454dca8bf8137e8cb634e25954a8bab Is it better now?